project.yaml

```diff
@@ -503,9 +503,10 @@ Layer:
       <<: *osm2pgsql
       table: |-
         (SELECT
-            way, landuse, leisure
+            way, COALESCE(landuse, CASE WHEN "natural" = 'wood' THEN 'forest' END) AS feature
```

Comment:
I would add an explicit `ELSE NULL` case. If not, I believe this will give an error whenever the WHERE part is extended. In fact I think the current code is equivalent to `COALESCE(landuse, 'forest')`.